tests/common.py

```diff
@@ -194,7 +194,7 @@ def hilbert_sort(conf, D):
     for xyz in int_confs.tolist():
         dist = hc.distance_from_coordinates(xyz)
         dists.append(dist)
-    perm = np.argsort(dists)
+    perm = np.argsort(dists, kind="stable")
```

Comment:
I guess the unstable sort didn't turn out to be the culprit after all (presumably the instability is deterministic?)

Reply:
It did not, but it seems like a nice change to have regardless.